core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedService.java

```diff
@@ -310,6 +313,11 @@ private HttpResponse serve0(ServiceRequestContext ctx, HttpRequest req) {
     private CompletionStage<HttpResponse> serve1(ServiceRequestContext ctx, HttpRequest req,
                                                  AggregationType aggregationType) {
         final CompletableFuture<AggregatedResult> f;
+        final AttributeKey<List<String>> PARAM_LIST_KEY = AttributeKey.valueOf(List.class, "names");
+        final List<String> names = resolvers.stream()
+                                            .map(AnnotatedValueResolver::httpElementName)
+                                            .collect(Collectors.toList());
+        ctx.setAttr(PARAM_LIST_KEY, names);
```

Comment:
- We could just pass the param list as an additional parameter of `aggregateMultipart()`.
- We could collect this information when creating an `AnnotatedService` instance, not every time service each request.